packages/anasazi/epetra/util/ModeLaplace/BlockPCGSolver.h

```diff
@@ -115,7 +115,7 @@ class ANASAZIEPETRA_MODELAPLACE_LIB_DLL_EXPORT BlockPCGSolver : public virtual E
     const char* Label() const { return "Epetra_Operator for Block PCG solver"; };
 
     bool UseTranspose() const { return (false); };
-    int SetUseTranspose(bool useTranspose) { return 0; };
+    int SetUseTranspose(bool /* useTranspose */) { return 0; };
```

Comment:
Silly me.  Of course this wouldn't work.  @trilinos/anasazi, would you prefer to remove `bool useTranspose` entirely, or would you rather throw a `(void)useTranspose` before the `return 0`?

Reply:
It really should return -1, per Epetra semantics:

https://github.com/trilinos/Trilinos/blob/1f66625b666138226fb6412fa19afcc35402b61d/packages/epetra/src/Epetra_Operator.h#L81

Reply:
I would prefer this:
```
int SetUseTranspose (bool /* useTranspose */ ) { return -1; }
```
The "cast to `void`" trick doesn't always suppress compiler warnings.

Reply:
Switched to that solution.  Retesting&hellip;